### PR TITLE
Add Sayantika's entry

### PR DESCRIPTION
**Pull request by cosimameyer**
I added Sayantika's entry (https://github.com/rladies/global-team/issues/4)

### CODE REVIEW

File: data/global_team/current/sayantika_banik.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Sayantika Banik",
+  "role": ["Abstract Review System"],
+  "start": "June 2023",
+  "img": {
+    "url": "sayantika_banik.png",
```
Comment (drmowinckels) [inline data/global_team/current/sayantika_banik.json:6]:
```suggestion
    "url": "sayantika_banik.png"
```

File: data/global_team/current/sayantika_banik.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    "credit": ""
```
Comment (drmowinckels) [inline data/global_team/current/sayantika_banik.json:7]:
```suggestion
```

**Review by drmowinckels — CHANGES_REQUESTED** [review]
Looks great! Just suggesting to remove the credit bit if there is not credit to be given :)

File: data/global_team/current/sayantika_banik.json (hunk 2 above)
Comment (cosimameyer) [reply data/global_team/current/sayantika_banik.json:7]: Excellent catch, thanks! :)

**Review by drmowinckels — APPROVED** [review]

**Pull request merged**